File: MedContactCore/DataTransferObjects/AppointmentDto.cs
using MedContactDb.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MedContactCore.DataTransferObjects
{
    public class AppointmentDto
    {
        public Guid Id { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime? StartTime { get; set; }
        public DateTime? EndTime { get; set; }
        public Guid? DayTimeTableId { get; set; }
        public DayTimeTableDto? DayTimeTable { get; set; }
        public Guid? CustomerDataId { get; set; }
      
    }
}

File: medcontact-angular/medcontact/src/app/models/appointmentdto.ts
import { Customerdatadto } from "./customerdatadto";
import { Daytimetabledto } from "./daytimetabledto";

export class Appointmentdto {
  id?: string;
  creationDate?: Date;
  startTime?: Date;
  endTime?: Date;
  dayTimeTableId?: string;
  DdyTimeTable?:  Daytimetabledto;
  customerDataId?: string;
  customerData?: Customerdatadto;
}
